raven_aiohttp.py

```diff
@@ -61,5 +80,8 @@ def f():
                     success_cb()
             except Exception as exc:
                 failure_cb(exc)
+            finally:
+                if not self.keepalive:
+                    yield from session.close()
 
-        asyncio.async(f(), loop=self._loop)
+        asyncio.ensure_future(f(), loop=self._loop)
```

Comment:
Mandatory `asyncio.ensure_future` adds backward incompatibility: `aiohttp` works with Python 3.4.1+ but `ensure_future` was added in 3.4.4 IIRC.

Please add workaround like https://github.com/KeepSafe/aiohttp/blob/master/aiohttp/helpers.py#L17-L20 
Don't import it from aiohttp: it's a private API. 
Just copy-paste.